README.md

```diff
@@ -56,51 +64,86 @@ Improtant files:
 
 # Part 2: Debugging linux kernel using qemu and gdb
 
-After we compiled linux kernel and rootfs we can debug it.
-Our emulator will be qemu because qemu is really lightweight emulator that can be easily conigured to run almost anything and qemu works fine with kvm which improves the performence.
+After we compiled the linux kernel and rootfs we can debug it.
+Our emulator will be qemu because qemu is a really lightweight emulator that can be easily configured to run almost anything and qemu works fine with kvm which improves the performance.
 
-First, we need to convert our raw rootfs to qemu format which will be later used for creating snapshot.
+8. First, we'll enable ssh connections by changing `sshd_config`
+```
+sudo -i
+cd /path/to/buildroot/output/images
+mkdir /mnt/dbg_kernel_fs
+mount rootfs.ext2 /mnt/dbg_kernel_fs
+echo "PermitRootLogin yes" >> /mnt/dbg_kernel_fs/etc/ssh/sshd_config
+umount /mnt/dbg_kernel_fs
+rmdir /mnt/dbg_kernel_fs
+exit
+```
 
-Note: rootfs.ext4 is just a symlink to rootfs.ext2
+Now we'll convert our raw rootfs to qemu format which will enable us to create snapshots later on.
 
-8. qemu-img convert -f raw -O qcow2 rootfs.ext2 rootfs.qcow2
+9. 
+```
+cd ./output/images
+qemu-img convert -f raw -O qcow2 rootfs.ext2 rootfs.qcow2`
+```
+
+> Note: rootfs.ext4 is just a symlink to rootfs.ext2
 
 ## Launch the vm
-9. sudo qemu-system-x86_64 -enable-kvm -cpu host -s -kernel bzImage  -m 2048 -hda rootfs.qcow2 -append "root=/dev/sda rw nokaslr" -net nic,model=virtio -net user,hostfwd=tcp::5555-:22
 
-Lets explain our command:
-* -enable-kvm -> kvm is a virtualization solution for linux which use hardware virtualization extensions, we will use it in order to improve the vm performence
+Buildroot has created a script at `./output/images/start-qemu.sh`. We'll
```

Comment:
I really don't like this buildroot dependecy...

Reply:
Good point. I've added `start-qemu.sh` into the repo